### ISSUE TITLE
Any method to export a static version of inter topic distance map for a paper?

### ISSUE THREAD

**Issue by cyriltw** [MaartenGr/BERTopic#1019]
The package is helpful and I love the ability to export figures to html for web. I'm trying to export content in a static form for a research paper. 

So is there a way to export the inter topic distance map to show the topics and the clusters as a png/pdf? 

![Screenshot 2023-02-19 at 10 20 52 PM](https://user-images.githubusercontent.com/9545140/220002105-227bb4b1-0b4e-4bcb-824a-7d51b8d2039c.png)

The figure generated through `topic_model.visualize_topics()` model. 

I tried the save figure through; 
```
fig = topic_model.visualize_topics()
fig.write_html("path/to/file.html")

```
but it was giving a static figure without the topics. 

Any ideas? Thanks

**Comment by MaartenGr** [comment]
Yes, the figure that is outputted is created with `plotly` and allows you to do static image exports as described [here](https://plotly.com/python/static-image-export/).

**Comment by cyriltw** [comment]
> Yes, the figure that is outputted is created with `plotly` and allows you to do static image exports as described [here](https://plotly.com/python/static-image-export/).

Hi @MaartenGr thanks for the response. I went through the page, but it didnt mention anything about exporting with the labels/text. Or am I missing something? 

I’m trying to export the figure to PDF (this can be done) but with the text (labels) for each cluster. So the words that shows up when you hover over a topic.

**Comment by MaartenGr** [comment]
It is not possible to also export the words that show up when you hover over a topic. This is because the hover-over is an interactive component and changes depending on where and how you hover over certain items. As such, these do not have a static place in the figure and therefore cannot be exported statically. Perhaps using `.visualize_documents` would be better suited for that.
